packages/components/helptext/README.mdx

```diff
@@ -1,28 +1,31 @@
+# @contentful/f36-helptext
```

Comment:
I saw that @suevalov didn't mention a few components in the [proposed components hierarchy for v4](https://github.com/contentful/forma-36/pull/745). I was wondering where to put ControlledInput & ControlledInputField. In the end, I believe we should move them to the place where they are originally used. Since the HelpText is used by ControlledInputField and other elements, I suggest moving it to `f36-forms`.

I'm currently working on that package but can't get it working as long as I have a back dependency to `f36-react-components` (which are HelpText & FormLabel). Please reach out to me, so we can discuss this :)

Reply:
not the biggest fan of deciding for the user if something is a form element or not(especially if its just a <p>), but sure - I can move it there.

Reply:
Discussed this via DM: Let's wait for a general decision.